pkg/republisher/uuid_collection_republisher.go

```diff
@@ -68,3 +69,7 @@ func (r *NotifyingUCRepublisher) RepublishUUIDFromCollection(uuid, tid string, c
 func extendTimeToLength(start time.Time, length time.Duration) {
 	time.Sleep(time.Duration(start.Add(length).UnixNano()-time.Now().UnixNano()) * time.Nanosecond)
 }
+
+func isVideoMetadata(originSystemID string, collectionName string) bool {
+	return strings.Contains(originSystemID, "next-video-editor") && collectionName == "video-metadata"
```

Comment:
Suggestion (non-blocking) Maybe for performance reason to swap the conditions, like:
```
return collectionName == "video-metadata" && strings.Contains(originSystemID, "next-video-editor")
```

Reply:
makes sense, will do :)